src/PageLayout/PageLayout.stories.tsx

```diff
@@ -0,0 +1,298 @@
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Pane',
+      defaultValue: {
+        summary: '"medium"'
+      }
+    }
+  },
+  'Pane.divider': {
+    type: {
+      name: 'enum',
+      value: ['none', 'line']
+    },
+    defaultValue: 'none',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Pane',
+      defaultValue: {
+        summary: '"none"'
+      }
+    }
+  },
+  'Pane.dividerWhenNarrow': {
+    type: {
+      name: 'enum',
+      value: ['inherit', 'none', 'line', 'filled']
+    },
+    defaultValue: 'inherit',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Pane',
+      defaultValue: {
+        summary: '"inherit"'
+      }
+    }
+  },
+  'Show footer?': {
+    type: 'boolean',
+    defaultValue: true,
+    table: {
+      category: 'Footer'
+    }
+  },
+  'Footer.divider': {
+    type: {
+      name: 'enum',
+      value: ['none', 'line']
+    },
+    defaultValue: 'none',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Footer',
+      defaultValue: {
+        summary: '"none"'
+      }
+    }
+  },
+  'Footer.dividerWhenNarrow': {
+    type: {
+      name: 'enum',
+      value: ['inherit', 'none', 'line', 'filled']
+    },
+    defaultValue: 'inherit',
+    control: {
+      type: 'radio'
+    },
+    table: {
+      category: 'Footer',
+      defaultValue: {
+        summary: '"inherit"'
+      }
+    }
+  }
+}
+
+Playground.args = {
+  containerWidth: 'xlarge',
+  outerSpacing: 'normal',
+  rowGap: 'normal',
+  columnGap: 'normal'
+}
+
+export const PullRequestPage = () => (
+  <PageLayout>
+    <PageLayout.Header>
+      <Box sx={{display: 'flex', flexDirection: 'column', gap: 3}}>
+        <Box>
+          <Heading as="h1" sx={{fontWeight: 'normal'}}>
+            Input validation styles <Text sx={{color: 'fg.muted', fontWeight: 'light'}}>#1831</Text>
+          </Heading>
+          <Box sx={{display: 'flex', gap: 2, alignItems: 'center'}}>
+            <StateLabel status="pullOpened">Open</StateLabel>
+            <Text sx={{fontSize: 1, color: 'fg.muted'}}>
+              <Link href="#" muted sx={{fontWeight: 'bold'}}>
+                mperrotti
+              </Link>{' '}
+              wants to merge 3 commits into <BranchName href="#">main</BranchName> from{' '}
+              <BranchName href="#">mp/validation-styles</BranchName>
+            </Text>
+          </Box>
+        </Box>
+        <TabNav>
+          <TabNav.Link href="#" selected>
+            Conversation
+          </TabNav.Link>
+          <TabNav.Link href="#">Commits</TabNav.Link>
+          <TabNav.Link href="#">Checks</TabNav.Link>
+          <TabNav.Link href="#">Files changed</TabNav.Link>
+        </TabNav>
+      </Box>
+    </PageLayout.Header>
+    <PageLayout.Content>
+      <Box sx={{border: '1px solid', borderRadius: 2, borderColor: 'border.default', height: 200}}></Box>
+    </PageLayout.Content>
+    <PageLayout.Pane>
```

Comment:
Should we make `position` a mandatory prop? If you flip the order of <Pane/> and <Content/>, you still get the same result which is confusing 🤔 Making `position` explicit would help?

Reply:
Hmm, I like that PageLayout works well out-of-the-box with no extra props but I understand what you're saying. Let's move forward with it as an optional prop and see if it causes confusion